challenges/platform/troubleshooting.md

```diff
@@ -19,44 +17,46 @@ The interview team and candidate will join a scheduled zoom call for the trouble
 - 15m - Wrap up, Q&A, etc.
 
 ## Areas of Focus
-We are not looking for any specific application knowledge. The test is designed to troubleshoot generic configuration or system faults on any software deployed to a generic Linux host. While this does not require any deployed application knowledge, some familiarity with collecting logs or investigating unfamiliar software is part of the process.
+Our test is designed to cover a range of configuration errors and system faults encountered on a Linux server. Successful candidates will be familiar with collecting logs from and investigating unfamiliar software in order to diagnose underlying system problems. We avoid challenges that require application specific expertise; Instead, we use applications as a jumping off point for diagnosing underlying system issues.
 
 The faults are present in:
 - kernel / systemd / cgroups
-- filesystem or volumes
+- filesystems and volumes
 - networking
 - TLS connectivity
-- Application behaviour
+- Application behavior
 
 We use the following software to represent applications on the host, which are experiencing faults:
 - nginx
 - etcd
 - docker
+- kubernetes
 - ipvs
 
-As this is meant to test the way we normally work, you may use any resource you would normally use when troubleshooting a system; for example, using google as a reference is normal and expected.
+While it is not required to be familiar with these applications, familiarity will lend a leg up when you need to querying logs or judging expected behavior.
+
+This is meant to test the way we normally work; please use any resource you would use when troubleshooting a system at home or in your job. For example, it is encouraged to Google if you encounter a promising, but unfamiliar error message. We look favorably on candidates who know how to formulate good queries when they're stumped.
+
+Similarly, you're welcome to install troubleshooting packages on the servers. While we've installed many common debugging tools, if you're used to having `htop`, `tldr`, or `execsnoop` at your fingertips, don't hesitate to install them.
 
 ## Pitfalls and Gotchas
-Some of the challenges exercise some of the hardest problems we've seen in the wild, so it's common to get stuck on one problem. If you do find you're not making progress on an issue, it's fine to move on and skip a problem and come back to it later.
+Some challenges reproduce the hardest problems we've seen in the wild, so it's common to get stuck. If you don't know where to start or find you're not making progress on an issue, please skip that problem.  You may return to it later, after you complete the other problems.
```

Comment:
personally I prefer this as a suggestion rather than an ask (i.e please feel free to move onto a different problem and come back to it if you find yourself stuck)

Reply:
4efe257 rephrases this to:

```
If you don't know where to start or find you're not making progress on an issue, feel free to move on to a later problem. You're welcome to return to skipped problems later.
```

Let me know if you'd like to see further tweaks.